libraries/chain/controller.cpp

```diff
@@ -848,308 +734,10 @@ struct controller_impl {
       resource_limits.add_indices();
    }
 
-   void add_contract_tables_to_snapshot( const snapshot_writer_ptr& snapshot ) const {
```

Comment:
Where is those snapshot code moved to? Just saw Jeff's PR https://github.com/EOSIO/eos/pull/9426 
 (removing DISK mode). He simplifies snapshot code. You can take a look there, to consolidate with his code.

Reply:
Thanks for bringing Jeff's PR to my attention. I think those changes were a mistake.